operator/templates/stateful-set.yaml

```diff
@@ -62,6 +62,13 @@ spec:
                 command:
                   - nodetool
                   - drain
+          readinessProbe:
+            exec:
+              command:
+                - nodetool status -p {{ .Params.JMX_PORT }}
```

Comment:
@mpereira 
What is the cassandra working dir? Prob failed (I tried different ways to run)
```
Readiness probe errored: rpc error: code = Unknown desc = failed to exec in container: failed to start exec "8ffdd0483521cc3b6dad2b461179285023ba852179c50c919dec6c2f915b2f6a": OCI runtime exec failed: exec failed: container_linux.go:345: starting container process caused "exec: \"/var/lib/cassandra/nodetool status -p 7199\": stat /var/lib/cassandra/nodetool status -p 7199: no such file or directory": unknown
```